modules/7.2.7/module.yaml

```diff
@@ -17,6 +17,9 @@ artifacts:
     - name: jboss-eap-7.2.7-patch
       target: jboss-eap-7.2.7-patch.zip
       md5: 34a8787d5f5cdf54d5c1477ab1bdf47b
+    - name: jbeap-18811
```

Comment:
This should be:

name: jboss-eap-7.2.7-patch
target: jboss-eap-7.2.7-patch.zip
md5: 34a8787d5f5cdf54d5c1477ab1bdf47b

Then you cekit-cache add https://url/to/file/jboss-eap-7.2.7-patch.zip --md5=34a8787d5f5cdf54d5c1477ab1bdf47b

Otherwise this will be hard to override in OSBS / CPaaS etc.